### PR TITLE
feat(ui5-breadcrumbs): add position numbering scheme "x of y"

### PR DESCRIPTION
**Pull request by dobrinyonkov**
Add position text to the accessibleNameRef property of the links.
Additionally add the text to the aria-label attribute of the currentLocation

Separate counting of the links outside the drop-down and the ones inside the drop-down.

### CODE REVIEW

File: packages/main/src/Breadcrumbs.js [hunk 1]
```diff
@@ -516,12 +538,43 @@ class Breadcrumbs extends UI5Element {
...
 			items.pop();
 		}
 
-		return items.filter(item => this._isItemVisible(item));
+		return items
+			.filter(item => this._isItemVisible(item))
+			.map((item, index) => {
+				const { accessibleNameRef, accessibleName } = this._getItemAccessibleNameData(item, index + 1, itemsCount);
+
+				item.accessibleNameRef = accessibleNameRef;
```
Comment (nnaydenow) [inline packages/main/src/Breadcrumbs.js:552]: lets be consistent and name this property _accessibleNameRef as other

File: packages/main/src/Breadcrumbs.js [hunk 2]
```diff
@@ -516,12 +538,43 @@ class Breadcrumbs extends UI5Element {
...
+				return item;
+			});
+	}
+
+	/**
+	 * Getter for accessible name of the current location. Includes the position of the current location and the size of the breadcrumbs
+	 */
+	get _currentLocationAccName() {
+		let items = this.getSlottedNodes("items").slice(this._overflowSize);
+		items = items.filter(item => this._isItemVisible(item));
```
Comment (nnaydenow) [inline packages/main/src/Breadcrumbs.js:564]: i saw that we are using doing this filter more than once. can we add new getter called visibleItems or something like that. this is not related to you change. it's just idea for refactoring

File: packages/main/src/Breadcrumbs.js [hunk 3]
```diff
@@ -516,12 +538,43 @@ class Breadcrumbs extends UI5Element {
...
+
+		const positionText = this._getItemPositionText(items.length, items.length);
+		const lastItem = items[items.length - 1];
+
+		if (!lastItem) {
+			return positionText;
+		}
+
+		if (lastItem.accessibleName) {
+			return `${lastItem.accessibleName} ${lastItem.innerText} ${positionText}`;
```
Comment (nnaydenow) [inline packages/main/src/Breadcrumbs.js:574]: for other items text is in following order => innerText, accessibleName, position and for current location => accessibleName, innerText, position. Which is the correct speech out put order?

File: packages/main/src/Breadcrumbs.hbs [hunk 4]
```diff
@@ -31,7 +35,7 @@
 
         {{#if _endsWithCurrentLocationLabel}}
         <li class="ui5-breadcrumbs-current-location" @click="{{../_onLabelPress}}">
-            <span id={{this._id}}-labelWrapper>
+            <span aria-label="{{_currentLocationAccName}}" role="link" id={{this._id}}-labelWrapper>
```
Comment (nnaydenow) [inline packages/main/src/Breadcrumbs.hbs:38]: can we add quotes around this id

File: packages/main/src/Breadcrumbs.js (hunk 1 above)
Comment (dobrinyonkov) [reply packages/main/src/Breadcrumbs.js]: AccessibleNameRef usage is removed now.

File: packages/main/test/specs/Breadcrumbs.spec.js [hunk 5]
```diff
@@ -210,11 +210,13 @@ describe("Breadcrumbs general interaction", () => {
 
 	it("renders accessible names of non-overflowing link items", async () => {
 		const breadcrumbs = await browser.$("#breadcrumbsWithAccName"),
-			link = (await breadcrumbs.shadow$$("ui5-link"))[3],
-			expectedAriaLabel = "last link acc name";
+			item = await browser.$("#lastItemWithACCName"),
+			itemId = await item.getProperty('_id'),
```
[the remark this answers is not in the document]
Comment (dobrinyonkov) [reply packages/main/test/specs/Breadcrumbs.spec.js]: Done

File: packages/main/src/themes/Breadcrumbs.css [hunk 6]
```diff
@@ -1,3 +1,5 @@
+@import "./InvisibleTextStyles.css";
```
[the remark this answers is not in the document]
Comment (dobrinyonkov) [reply packages/main/src/themes/Breadcrumbs.css]: Done

**Review by nnaydenow — APPROVED** [review]

**Review by nnaydenow — APPROVED** [review]
